roofit/roofitcore/test/CMakeLists.txt

```
@@ -24,6 +24,7 @@ ROOT_ADD_GTEST(testRooAbsCollection testRooAbsCollection.cxx LIBRARIES RooFitCor
 ROOT_ADD_GTEST(testRooDataSet testRooDataSet.cxx LIBRARIES Tree RooFitCore)
 ROOT_ADD_GTEST(testRooFormula testRooFormula.cxx LIBRARIES RooFitCore)
 ROOT_ADD_GTEST(testRooProdPdf testRooProdPdf.cxx LIBRARIES RooFitCore RooFit)
+ROOT_ADD_GTEST(AutomaticDiffSimple AutomaticDiff/simple.cpp LIBRARIES RooFitCore RooFit)
```

```suggestion
ROOT_ADD_GTEST(testAutomaticDiffSimple AutomaticDiff/simple.cpp LIBRARIES RooFitCore RooFit)
```
The convention is that all the binaries start with `test`